modules/serverless-install-kafka-odc.adoc

```diff
@@ -21,10 +21,25 @@ spec:
         bootstrapServers: <bootstrap_servers> <2>
     source:
         enabled: true <3>
+    broker:
+        enabled: true <4>
+        defaultConfig:
+            bootstrapServers: <bootstrap_servers> <5>
+            numPartitions: <num_partitions> <6>
+            replicationFactor: <replication_factor> <7>
 ----
 <1> Enables developers to use the `KafkaChannel` channel type in the cluster.
 <2> A comma-separated list of bootstrap servers from your AMQ Streams cluster.
 <3> Enables developers to use the `KafkaSource` event source type in the cluster.
+<4> Enables developers to use the Knative Kafka broker implementation in the cluster.
+<5> A comma-separated list of bootstrap servers from your Red Hat AMQ Streams cluster.
```

Comment:
Should we highlight that this can be a `Red Hat OpenShift Streams for Apache Kafka` cluster as well?

Reply:
Yeah, I'd leave it out for now, but I will definitely come back to you on that @pierDipi !